examples/vision/text_image_denoiser.py

```diff
@@ -0,0 +1,299 @@
+"""
+Title: Text Image Denoiser as OCR Preprocessor.
+Author: [Anish B](https://twitter.com/anishhacko)
+Date created: 2021/01/18
+Last modified: 2022/01/27
+Description: Example of Text Image Denoiser coupled with Tesseract OCR engine to improve text readability.
+"""
+"""
+## Introduction
+
+This Example explains a simple **Text Image Denoiser for OCR** using U-Net based
+Architectutre. **Autoencoders** are mostly employed for **Image Restoration** problems
```

Comment:
The text copy should be improved, it contains many typos, etc. Please find someone who could help you with it.